inc/Engine/Optimization/Minify/JS/Combine.php

```diff
@@ -194,6 +193,13 @@ function( $script ) {
 							}
 						}
 
+						if ( ! empty( $this->jquery_urls ) ) {
+							$jquery_urls = implode( '|', $this->jquery_urls );
+							if ( preg_match( '#^(' . $jquery_urls . ')$#', rocket_remove_url_protocol( $matches['url'] ) ) ) {
+								return;
+							}
+						}
```

Comment:
What about exclusion of the internal jquery? If the check is performed here only, it won't be excluded

Reply:
Yup. This is responsible only for external jquery exclusion.

Internal jquery exclusion is handled in here: `is_minify_excluded_file()`